charts/nextcloud/values.yaml

```diff
@@ -465,6 +477,14 @@ metrics:
 
   # podLabels: {}
 
+  nginx:
+    enabled: false
+    allow:
+    - 10.233.105.0/24
+    - 10.43.0.0/16
```

Comment:
What are those?

Reply:
Those serve just as an example (default k3s installation networks). I can comment them out if preferred.

Reply:
Yeah I think that's better